### PR TITLE
[GS-1957] Added step to remove user or group in tenant

### PR DESCRIPTION
**Pull request by amateos-stratio**
Added step to remove user or group in tenant

### CODE REVIEW

File: src/main/java/com/stratio/qa/specs/RestSpec.java [hunk 1]
```diff
@@ -1180,4 +1180,67 @@ public void updateResource(String resource, String resourceId, String loginInfo,
         }
     }
 
+
+    @When("^I remove '(user|group)' '(.+?)' in tenant '(.+?)'$")
```
Comment (aalfonso-stratio) [inline src/main/java/com/stratio/qa/specs/RestSpec.java:1184]: yo pondría:
^I remove '(user|group)' '(.+?)' from tenant '(.+?)'$

Comment (amateos-stratio) [reply src/main/java/com/stratio/qa/specs/RestSpec.java]: changed :)

**Comment by pbedia-stratio** [comment]
LGTM

**Comment by amateos-stratio** [comment]
merged into master

**Pull request merged**